README.md

```diff
@@ -173,6 +173,54 @@ $ bundle exec fastlane alpha
 react-native run-ios --configuration Release --device 'YOUR_DEVICE_NAME'
 ```
 
+### Sviluppo con App Backend e IDP di test in locale
+
+Per sviluppare l'applicazione utilizzando in locale l'App Backend e un IDP di test, è necessario seguire alcuni step aggiuntivi descritti di seguito.
+
+#### Installazione di App Backend e IDP di test
+
+Seguire la documentazione del repository [italia-backend](https://github.com/teamdigitale/italia-backend).
+
+#### WebView, HTTPS e ceriticati autofirmati
+
+Allo stato attuale react-native non consente di aprire WebView su url HTTPS con certificato autofirmato. L'IDP di test però fa utilizzo di HTTPS e di un certificato autofirmato. Per ovviare a questo problema è possibile installare in locale un Proxy che faccia da proxy-pass verso l'App Backend e l'IDP.
+
+##### Installazione di mitmproxy
+
+Un proxy semplice da utilizzare e addatto al nostro scopo è [mitmproxy](https://mitmproxy.org/). Per l'installazione seguire la [pagina di documentazione](https://docs.mitmproxy.org/stable/overview-installation/) del sito sito ufficiale.
+
+Una volta terminata l'installazione creare un file chiamato `mitmproxy_metro_bundler.py` con il seguente contenuto:
```

Comment:
mettiamo questo file nel repo dell'app, sotto la directory `contrib` (da creare), e in questa documentazione diciamo che il file si trova li